flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/dialect/JdbcDialect.java

```diff
@@ -31,7 +31,7 @@
 /**
  * Handle the SQL dialect of jdbc driver.
  */
-public interface JDBCDialect extends Serializable {
+public interface JdbcDialect extends Serializable {
```

Comment:
How about to add an `@Internal` annotation on this to tell this is not ready to be exposed to users.